cmd/delimiter.go

```diff
@@ -32,7 +32,7 @@ func (sub *DelimiterSubcommand) Run(args []string) {
 }
 
 func ChangeDelimiter(inputCsv *InputCsv, inputDelimiter, outputDelimiter string) {
-	inputDelimiterRune := GetDelimiterFromString(inputDelimiter)
+	inputDelimiterRune := GetDelimiterFromStringOrPanic(inputDelimiter)
```

Comment:
Changing to ...OrPanic here negates the effort of the next block to test for a zero-length delimiter and set the delimiter to the default comma.  This effectively breaks the previous default behavior of the delim subcommand which did not require either being set: now, both must be explicitly set.

Also, using ...OrPanic here hides which delimiter, input or output?, has the problem as it just provides a generic error without any context.

All of that added up to some confusion for me as I was trying to diagnose why a previously OK script was now failing.